packages/nextjs/next.config.mjs

```diff
@@ -27,6 +29,11 @@ const nextConfig = {
   webpack: (config) => {
     config.resolve.fallback = { fs: false, net: false, tls: false };
     config.externals.push("pino-pretty", "lokijs", "encoding");
+    config.plugins.push(
+      new webpack.NormalModuleReplacementPlugin(/^node:(.*)$/, (resource) => {
+        resource.request = resource.request.replace(/^node:/, "");
+      }),
+    );
```

Comment:
just curious why we need this webpack extension ser?

Reply:
I found there is a module `node.url` can not be parsed correctly, so add this plugin to handle this case